src/tooltip/Tooltip.js

```diff
@@ -0,0 +1,13 @@
+const { default: styled } = require("styled-components");
```

Comment:
Any reason we're using `require` instead of `import` here?

Reply:
oh jeez i didn't even notice this ... my text editor must have gone rogue